src/main.cc

```diff
@@ -783,6 +580,9 @@ int main(int argc, char** argv)
     uncore.LLC.llc_initialize_replacement();
     uncore.LLC.llc_prefetcher_initialize();
 
+    // initialize virtual memory system
+    vmem = new VirtualMemory(NUM_CPUS, 8589934592, 4096, 4, 1);
```

Comment:
Here, we have a hard-coded constant as the RNG seed. Is this to achieve deterministic shuffling? If so, why is that desirable?

Reply:
Everything about ChampSim is deterministic.  I don't want any part of ChampSim to ever deviate from that.